utils/helpers.py

```diff
@@ -88,11 +88,9 @@ def get_asset_memoized(*, kili, project_id, first, skip):
 def asset_is_kept(asset, labeling_statuses: List[str] = ["LABELED", "UNLABELED"]) -> bool:
     labeled = len(asset["labels"]) > 0
     unlabeled = len(asset["labels"]) == 0
-    if "LABELED" in labeling_statuses:
-        return labeled
-    if "UNLABELED" in labeling_statuses:
-        return unlabeled
-    return False
+    return ("LABELED" in labeling_statuses and labeled) or (
+        "UNLABELED" in labeling_statuses and unlabeled
+    )
```

Comment:
sorry for that

Reply:
no pb, I validated it. We will ramp up on testing :-)